src/uwtools/cli.py

```diff
@@ -636,16 +649,17 @@ def _abort(msg: str) -> None:
     sys.exit(1)
 
 
-def _add_args_quiet_and_verbose(group: Group) -> ActionChecks:
+def _add_args_verbosity(group: Group) -> SubmodeChecks:
```

Comment:
Is `SubmodeChecks` actually still defined somewhere? I thought it was eliminated in my previous PR...